pipelines/controller/src/main/java/com/google/fhir/analytics/HiveTableManager.java

```diff
@@ -48,14 +47,11 @@ public class HiveTableManager {
 
   private static final String THRIFT_CONTAINER_PARQUET_DIR = "/dwh";
 
-  public HiveTableManager(DatabaseConfiguration hiveDbConfig, String viewsDir)
-      throws PropertyVetoException {
+  public HiveTableManager(DatabaseConfiguration hiveDbConfig, String viewsDir) {
     this.dataSource =
         JdbcConnectionPools.getInstance()
             .getPooledDataSource(
-                JdbcConnectionPools.dbConfigToDataSourceConfig(hiveDbConfig),
-                CONNECTION_POOL_SIZE,
-                CONNECTION_POOL_SIZE);
+                JdbcConnectionPools.dbConfigToDataSourceConfig(hiveDbConfig), CONNECTION_POOL_SIZE);
```

Comment:
Instead of passing `CONNECTION_POOL_SIZE` here, may be pick the value from `JdbcConnectionPools.MIN_CONNECTIONS` and add a comment here saying 'this is to make the minimum and maximum sizes are same so that connection resets are avoid to prevent the thrift error`